avocado/utils/distro.py

```diff
@@ -407,17 +459,21 @@ def register_probe(probe_class):
 register_probe(OpenEulerProbe)
 
 
-def detect():
+def detect(session=None):
     """
-    Attempts to detect the Linux Distribution running on this machine
+    Attempts to detect the Linux Distribution running on this machine.
+    If given an avocado.utils.ssh.Session object, it will attempt to detect the
```

Comment:
Please, insert an empty line between the short description and summary paragraph to comply with PEP257.